src/features/defi/providers/univ2/components/UniV2Manager/Deposit/components/Status.tsx

```diff
@@ -45,36 +39,41 @@ export const Status: React.FC<StatusProps> = ({ accountId }) => {
   const translate = useTranslate()
   const { state, dispatch } = useContext(DepositContext)
   const { query, history: browserHistory } = useBrowserRouter<DefiQueryParams, DefiParams>()
-  const { chainId } = query
+  const { chainId, assetNamespace, assetReference } = query
 
   const accountAddress = useMemo(
     () => (accountId ? fromAccountId(accountId).account : null),
     [accountId],
   )
 
-  const foxEthLpOpportunityFilter = useMemo(
+  const lpAssetId = toAssetId({ chainId, assetNamespace, assetReference })
+
+  const earnUserLpOpportunityFilter = useMemo(
     () => ({
-      lpId: foxEthLpAssetId,
-      assetId: foxEthLpAssetId,
+      lpId: lpAssetId as LpId,
+      assetId: lpAssetId,
       accountId,
     }),
-    [accountId],
+    [accountId, lpAssetId],
   )
-  const foxEthLpOpportunity = useAppSelector(state =>
-    selectEarnUserLpOpportunity(state, foxEthLpOpportunityFilter),
+  const earnUserLpOpportunity = useAppSelector(state =>
+    selectEarnUserLpOpportunity(state, earnUserLpOpportunityFilter),
   )
 
+  const assetId0 = earnUserLpOpportunity?.underlyingAssetIds[0] ?? ''
```

Comment:
can do a fancy one line destructure here if you feel that way inclined

Reply:
she sad :(
<img width="726" alt="image" src="https://user-images.githubusercontent.com/17035424/228026884-f51c803b-b7ec-40dc-924e-7be494a97b35.png">